src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -216,8 +216,6 @@ private void RestorePackages()
                 TraceWriter.Error(ex.Message, ex is CompilationErrorException ? null : ex);
 
                 startedEvent.Success = false;
-                TraceWriter.Error(ex.Message, ex);
```

Comment:
Fabio pointed out this inadvertant extra log that was introduced recently. It's currently causing duplicate logging :(